mark-griffith/short-response.md

```diff
@@ -0,0 +1,85 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer});
+   ```
+      Nothing because its missing a `
+
+      But seriously this code first initializes and declares a variable of singer with the string
```

Comment:
Assuming my typo was fixed. This answer is incorrect. Run this in the console and see what you get. Hint: "scope"